src/pages/index.js

```diff
@@ -283,26 +257,14 @@ class Home extends Component {
               />
               <section css={sectionStyles}>
                 <div id="examples">
-                  {examples.map((example, index) => (
-                    <div
+                  {examples.edges.map(({node}, index) => (
+                    <CodeExample
                       key={index}
-                      css={{
-                        marginTop: 40,
-
-                        '&:first-child': {
-                          marginTop: 0,
-                        },
-
-                        [media.greaterThan('xlarge')]: {
-                          marginTop: 80,
-                        },
-                      }}>
-                      <h3 css={headingStyles}>{example.title}</h3>
-                      <div
-                        dangerouslySetInnerHTML={{__html: example.content}}
-                      />
-                      <div id={example.id} />
```

Comment:
Whoops. Missed this. It doesn't _really_ work on the current iteration anyways, so not sure how valuable it is?

https://reactjs.org/#a-simple-component

Reply:
@bvaughn ugh, doesn't work in Firefox, works in Chrome! I'll add back the id!